includes/wizards/audience/class-audience-content-gates.php

```diff
@@ -140,11 +151,34 @@ public function register_api_endpoints() {
 			'/wizard/' . $this->slug,
 			[
 				'methods'             => 'GET',
-				'callback'            => [ $this, 'get_gates' ],
+				'callback'            => [ $this, 'get_config' ],
 				'permission_callback' => [ $this, 'api_permissions_check' ],
 			]
 		);
 
+		register_rest_route(
+			NEWSPACK_API_NAMESPACE,
+			'/wizard/' . $this->slug . '/config',
+			[
+				'methods'             => 'POST',
+				'callback'            => [ $this, 'update_config' ],
+				'permission_callback' => [ $this, 'api_permissions_check' ],
+				'args'                => [
+					'config' => [
+						'type'       => 'object',
+						'properties' => [
+							'countdown_banner' => [
+								'type' => 'object',
+							],
+							'content_gifting'  => [
+								'type' => 'object',
+							],
+						],
+					],
+				],
+			]
+		);
```

Comment:
I think this would be better maintained as separate endpoints, for each feature.

Reply:
0db065073 separates these into separate endpoints.